tutorials/java/webflux-contribution/tutorial.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Fixing array type look-up on r2dbc-postgresql, a reactive library from Spring Framework
+published: 2020-07-08
+tags: java, spring, webflux, r2dbc-postgresql
+language: java
+author-name: José Luis León
+twitter-profile: JoseLeonT
+github-profile: JoseLion
+description: Contributing to r2dbc-postgresql open-source library, the POstgreSQL implementation of the R2DBC project. The reactive alternative to JDBC that let us work on Spring WebFlux with relational databases.
```

Comment:
In order to keep the  introduction's ideas, I suggest the following description instead: 

The programming world is adopting practices from the functional and reactive programming paradigms. Here, we’ll review how we can integrate a relational database like PostgreSQL with Spring WebFlux while keeping the reactive design. Then we’ll see how we contributed to `r2dbc-postgresql` to solve a datatype decoding issue related to array-like types.

What do you think?

Reply:
I agree! 👍